### PR TITLE
fix: add two new interns

### PR DESCRIPTION
**Pull request by paulstey**
This PR adds a couple of our new interns, and we are also removing two interns that are no longer with us. 

#### Check the types of changes present in this PR:
- [ ] :bug: Bug
- [ ] :dragon: Feature
- [x] :frog: Data (data folder - people/opportunities)
- [ ] :dog: Content (content folder)
- [ ] :blowfish: Other. Specify:

#### Checklist:
- [x] Commitizen was used for all commits.
- [x] Local site looks as expected after changes.
- [x] Contributing guidelines were followed.
- [x] If changes affect development, was the README updated?

##### If PR to `production`
- [ ] Development site (https://datasci.brown.edu) was checked and looks as expected.

### CODE REVIEW

**Comment by mcmcgrath13** [comment]
@paulstey can you also remove the folks who are no longer interns?

File: data/people.yaml [hunk 1]
```diff
@@ -342,3 +322,23 @@
...
+  subteam: Institutional Data
+  title: Data Science Intern
+  github_username: quicklearnerjustin
+  brown_directory_uuid: 1034dc39-54c6-4b0b-aa2f-770783dedfc2
+  bio:
+  image: justin_main.jpg
+- name: Maitreya Venkataswamy
+  weight:
+  type: CCV Interns
+  team: Institutional Data
```
Comment (mcmcgrath13) [inline data/people.yaml]: team should be `Advanced Research Computing`

Comment (paulstey) [reply data/people.yaml]: > @paulstey can you also remove the folks who are no longer interns?

Thanks, @mcmcgrath13! Can you tell me to whom you are referring?

Comment (mcmcgrath13) [reply data/people.yaml]: Aryan and Tiffany.  Is Minsik still with the HPC group?

**Comment by fernandogelin** [comment]
@paulstey Would you mind also adding Ashley's intern:
- name: Yadi Yang 
- github: skyyaya28
IMG_6713
IMG_4318

**Comment by paulstey** [comment]
> @paulstey Would you mind also adding Ashley's intern:

Done! Thank you, @fernandogelin!!!

File: data/people.yaml (hunk 1 above)
Comment (paulstey) [reply data/people.yaml]: Done! Minsik is still with us, but currently out of the country.

**Comment by paulstey** [comment]
Are we okay to merge this?

**Review by mcmcgrath13 — APPROVED** [review]

**Review by fernandogelin — APPROVED** [review]

**Pull request merged**